### PR TITLE
Ingest PS1_STRM Catalog for Phot_z Estimates & Classification

### PR DESCRIPTION
**Pull request by AndyTza**
This pull request will ingest to kowalski the phot_z estimates and object morphological classification from the PS1-Source Types and Redshifts with Machine Learning catalog [(PS1-STRM)](https://archive.stsci.edu/hlsp/ps1-strm). 

See original reported issue #55

### CODE REVIEW

**Review by dmitryduev — CHANGES_REQUESTED** [review]
Very nice, @AndyTza! Looks like you need to install our pre-commit hook:
```
pip install pre-commit
pre-commit install
```
It'll do the linting for you and the CI will become green.

Otherwise, I think it's ok to get this in. Moving forward, I'm planning to write a general configurable "catalog ingester" with well-defined testing procedures, but that'll take some time.

**Comment by dmitryduev** [comment]
@AndyTza just updated the developer guidelines, please take a look: https://github.com/dmitryduev/kowalski#developer-guidelines

**Comment by AndyTza** [comment]
Thank you @dmitryduev 🚀  I ran the ```pip install pre-commit```  and ```pre-commit install``` and it looks like it passed lint now, but now failed at the `test_tns_watcher.py` test

**Comment by dmitryduev** [comment]
@AndyTza there were several issues with the code, which I committed to your branch (hope you don't mind :)

File: config.defaults.yaml [hunk 1]
```diff
@@ -253,6 +253,17 @@ kowalski:
             Teff: 1
             eTeff: 1
 
+        PS1_STRM:
+          filter: {}
+          projection:
+            _id: 1
```
Comment (dmitryduev) [inline config.defaults.yaml:259]:
```suggestion
            _id: 1
            objID: 1
```

File: config.defaults.yaml [hunk 2]
```diff
@@ -253,6 +253,17 @@ kowalski:
...
+            coordinates.radec_str: 1
+            class: 1
+            prob_Galaxy: 1
+            prob_Star: 1
```
Comment (dmitryduev) [inline config.defaults.yaml:263]:
```suggestion
            prob_Star: 1
            prob_QSO: 1
```

**Review by dmitryduev — APPROVED** [review]

**Pull request merged**